### ISSUE TITLE
Error at startup now

### ISSUE THREAD

**Issue by ohmichael** [SIMKL/script.simkl#74]
I’m using kodi 20.5 fork nothing wrong with the fork but ever since this morning/last night 4/14-4/15 I have been experiencing errors and I even tried relogging in by the settings and now it doesn’t let me when it allowed me to months ago. I think there’s a problem/issue with the kodi plugin for Simkl. I thought you should be aware. Let me know if it’s fixed because I did remove it as it was malfunctioning and giving me errors so I didn’t want it to cause a crash. Thanks for everything and hope you can fix it. I’m sure you can see the errors with all the information I provided. I don’t send logs. Need any verbal info on what I experienced please reply and I’ll reply accordingly. I’ve had problems with logs being sniffed out by others snooping around besides yourself so I don’t show logs publicly. I either do it in private or email. For safety concerns. But I’m sure you got enough to go on. Thanks, Michael.

**Comment by ohmichael** [comment]
Yes, will do, let me reinstall and if it does same thing you’ll get an email.
Thanks again, Michael

**Comment by ohmichael** [comment]
Closing ticket I reinstalled and it somehow let me relogin when earlier minutes before I made this ticket it wouldn’t let me but now it has and it appears to greet me on startup when before it would error out in the side panel/box itself.
Thanks again for the application, Simkl.
Been a lifesaver better than trakt.
Looks like I don’t have to send a log now even though I was prepared to make one.
Sorry about the extra comments I made but I went ahead and explained myself because I got into with some hardheaded devs back in the day and so I now get ahead of myself now so I don’t tread into same scenario again.
Thanks again, closing ticket, Michael.
Can never be too careful - people snooping all over.
